src/azure-cli-core/azure/cli/core/commands/client_factory.py

```diff
@@ -194,6 +194,16 @@ def _get_mgmt_service_client(cli_ctx,
         client_kwargs.update(_prepare_client_kwargs_track2(cli_ctx))
         client_kwargs['credential_scopes'] = resource_to_scopes(resource)
 
+        # Track 2 currently lacks the ability to take external credentials.
+        #   https://github.com/Azure/azure-sdk-for-python/issues/8313
+        # As a temporary workaround, manually add external tokens to 'x-ms-authorization-auxiliary' header.
+        #   https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/authenticate-multi-tenant
+        if getattr(cred, "_external_tenant_token_retriever", None):
+            _, _, _, external_tenant_tokens = cred.get_all_tokens(*resource_to_scopes(resource))
```

Comment:
What is `*`

Reply:
See [More on Defining Functions](https://docs.python.org/3.9/tutorial/controlflow.html#more-on-defining-functions).